packages/shared/src/components/modals/report/ReportPostModal.tsx

```diff
@@ -26,14 +26,14 @@ interface OptionalProps {
 }
 
 export type ReportedCallback = (
-  post: Post | PostBootData,
+  post: Post | PostBootData | ReadHistoryPost,
```

Comment:
No real action point but this is getting messy, I wonder if we can uniform them to one type that way potentially you could also have generic useVote hook and other interactions.

Reply:
Agreed, looked at it but currently a lot of tracking hooks have a wide requirements for post type so this is where the problem lies. We need to use `Pick` as much as we can to only require what is needed. Something for a backlog or future refactor for sure. 👍